kernel/src/net/iface/ext.rs

```diff
@@ -1,78 +1,13 @@
 // SPDX-License-Identifier: MPL-2.0
 
-use alloc::string::String;
-use core::sync::atomic::{AtomicU64, Ordering};
+use super::sched::PollScheduler;
+use crate::net::socket::ip::{datagram::DatagramObserver, stream::StreamObserver};
 
-use ostd::sync::WaitQueue;
+pub struct Ext;
```

Comment:
It may be better to rename the struct as `DummyExt` to distinguish the struct from the trait.